pallets/oracle-data-collection/src/lib.rs

```diff
@@ -460,17 +541,58 @@ pub mod types {
 		}
 	}
 
+	/// Information of a collection
+	#[derive(
+		Encode, Decode, PartialEq, Eq, Clone, TypeInfo, RuntimeDebugNoBound, MaxEncodedLen,
+	)]
+	#[scale_info(skip_type_params(T))]
+	pub struct CollectionInfo<T: Config> {
+		/// Maximum duration to consider an oracle value non-outdated.
+		/// An oracle value is consider updated if its timestamp is higher
+		/// than `now() - value_lifetime`
+		pub value_lifetime: Option<T::Timestamp>,
+
+		/// Minimun number of feeders to succesfully aggregate a value.
+		pub min_feeders: u32,
+	}
+
+	impl<T: Config> Default for CollectionInfo<T> {
+		fn default() -> Self {
+			Self {
+				value_lifetime: None,
+				min_feeders: 0,
```

Comment:
Lets use `min_feeders` = 1 here, otherwise, this will be a problem on the aggregation I would ugess, right?

Reply:
- If `0`, it means that the collection may not have all the expected keys. It's more general.
- If `1`, it means that the collection must have all expected keys.

The current built-in aggregation, with `0`, errors out with `KeyNotInCollection` because it can not aggregate with an empty list, but other `Aggregation` implementations could exist, setting a default value or similar if the list is empty.

For our concern, both `0` and `1` mean the same; it will give an error if there are no values for a key.

Reply:
BTW, good eye asking this!

Reply:
Perfect! Then that's super fine.